tests/super_test.py

```diff
@@ -129,3 +129,25 @@ def test_fix_super_noop(s):
 )
 def test_fix_super(s, expected):
     assert _fix_py3_plus(s) == expected
+
+
+@pytest.mark.xfail(
+    sys.version_info < (3, 5),
+    reason='async introduced in python 3.5',
+)
+@pytest.mark.parametrize(
+    ('s', 'expected'),
+    (
+        pytest.param(
+            'class C:\n'
+            '    async def foo(self):\n'
+            '        super(C, self).foo()\n',
+            'class C:\n'
+            '    async def foo(self):\n'
+            '        super().foo()\n',
+            id='super inside async func def',
+        ),
+    ),
+)
```

Comment:
probably don't need parametrize if there's only one test

Reply:
Yeah, right. I've forgotten to remove the old test btw 🤦‍♂

Reply:
Addressed in 56e91882b1c626d72c97882f0352d21ae93ae0fb.